code/modules/mob/mob_movement.dm

```diff
@@ -470,8 +470,9 @@
 		var/mob/living/silicon/ai/AI = src
 		if(AI.eyeobj)
 			var/turf/T = SSmapping.get_turf_above(get_turf(AI.eyeobj))
-			var/area/X = get_area(T) //Check theyre ON THE STATION and not trying to move down into the overmap
-			if(T && !istype(X, /area/space) && !istype(T, /turf/open/space/basic))
+			if(AI.overmap_ship) //So that they don't get an overmap camera on the ship Z-level.
+				return FALSE

```

Comment:
The overmap AI cameras shouldn't even have this verb. As AIs handle this differently, I believe. This might be cause for `CRASH`

Reply:
It might be possible via the helm console. We probably havent investigated that aspect.

Reply:
I told him to include this as a failsafe. overmap_ship is defined on mob itself IIRC